src/docker-it/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM ubuntu:14.04
+
+RUN apt-get install -y ssh && mkdir /var/run/sshd && chmod 0755 /var/run/sshd
```

Comment:
@vkuchyn please always run `apt-get update` in the same `RUN` that you run an `apt-get install` in, see [here](https://docs.docker.com/engine/userguide/eng-image/dockerfile_best-practices/) (section on apt-get) for the why.